src/components/Viewer.tsx

```diff
@@ -69,10 +83,40 @@ export default function Viewer({ handleTimeChange }: ViewerProps): ReactNode {
         }
     }, [page, setViewportToContainerSize, heightResized, trajectoryName]);
 
+    useEffect(() => {
+        const el = container.current;
+        el?.addEventListener("mousedown", (event) => {
+            if (!userHasInteracted) {
+                setFirstMousePosition({
+                    x: event.clientX,
+                    y: event.clientY,
+                });
+            }
+        });
+        container.current?.addEventListener("mouseup", (event) => {
+            if (
+                firstMousePosition.x !== event.clientX &&
+                firstMousePosition.y !== event.clientY
+            ) {
+                setUserHasInteracted(true);
+            }
+        });
```

Comment:
A little silly, but I think it would be possible for someone to click, drag, and return their mouse to the same position as it started (`firstMousePosition`) and have the hint overlay not go away. Probably a non-issue

Reply:
It would be incredible